### PR TITLE
Ensure that logger writes to stdout

### LINKED ISSUE #675: Log information writes to wrong output
### Summary
Output normally written to stdout is being written to `~/.ecr/log/ecr-login.log` if the directory `~/.ecr/log/` exists. This is due to a change at `52300fef` that switched the logging setup to use the default logger in the amazon-ecr-credential-helper.

### Description
```
~/go/src/github.com/aws/amazon-ecs-cli((HEAD detached at 52300fe))$ ecs-cli up
# no output
~/go/src/github.com/aws/amazon-ecs-cli((HEAD detached at 52300fe))$ echo $? 
1
~/go/src/github.com/aws/amazon-ecs-cli((HEAD detached at 52300fe))$ cat ~/.ecr/log/ecr-login.log
time="2018-11-28T21:46:19-08:00" level=fatal msg="Error executing 'up': You must either specify a custom role with the '--instance-role' flag or set the '--capability-iam' flag"
```

### Expected Behavior
The error message should print to stdout

### PR DESCRIPTION
**Pull request by SoManyHs**
Fixes #675 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by SoManyHs** [comment]
Even with existence of `~/.ecr/log`:
```
~/go/src/github.com/aws/amazon-ecs-cli(fix-logger)$ ls ~/.ecr/log/
ecr-login.log
```
Log info still writes to stdout at Info level:
```
~/ecs/ecs-cli/testing/projects/test_fields(master)$ ecs-cli compose up
INFO[0000] Using ECS task definition                     TaskDefinition="test_fields:64"
INFO[0000] Couldn't run containers                       reason="RESOURCE:PORTS"
INFO[0000] Couldn't run containers                       reason="RESOURCE:PORTS"

~/ecs/ecs-cli/testing/projects/test_fields(master)$ ecs-cli up
FATA[0000] Error executing 'up': You must either specify a custom role with the '--instance-role' flag or set the '--capability-iam' flag 
```

**Comment by SoManyHs** [comment]
Logging also correct for ECR-related commands:
```
~/go/src/github.com/aws/amazon-ecs-cli(fix-logger)$ ecs-cli push scorekeep-frontend
INFO[0000] Getting AWS account ID...                    
INFO[0000] Tagging image                                 image=scorekeep-frontend repository=xxxxxxxxxx51.dkr.ecr.us-west-2.amazonaws.com/scorekeep-frontend tag=
INFO[0000] Image tagged                                 
INFO[0000] Pushing image                                 repository=794715269151.dkr.ecr.us-west-2.amazonaws.com/scorekeep-frontend tag=
INFO[0001] Image pushed                                 
```

**Comment by SoManyHs** [comment]
Logging also correct for ECR-related commands:
```
~/go/src/github.com/aws/amazon-ecs-cli(fix-logger)$ ecs-cli push scorekeep-frontend:latest
INFO[0000] Getting AWS account ID...                    
INFO[0000] Tagging image                                 image=scorekeep-frontend repository=xxxxxxxxxx51.dkr.ecr.us-west-2.amazonaws.com/scorekeep-frontend tag=latest
INFO[0000] Image tagged                                 
INFO[0000] Pushing image                                 repository=xxxxxxxxx51.dkr.ecr.us-west-2.amazonaws.com/scorekeep-frontend tag=latest
INFO[0001] Image pushed                                 
```

File: ecs-cli/modules/utils/logger/logger.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"os"
+	"github.com/sirupsen/logrus"
+)
+
+func SetupLogger() {
+	logrusConfig()
+}
+
+func logrusConfig() {
+	logrus.SetLevel(logrus.InfoLevel)
```
Comment (PettitWesley) [inline ecs-cli/modules/utils/logger/logger.go]: Does this in any way conflict with: https://github.com/aws/amazon-ecs-cli/blob/master/ecs-cli/modules/app.go#L25

Can you test with `--debug` to check?

Comment (SoManyHs) [reply ecs-cli/modules/utils/logger/logger.go]:
```
~/go/src/github.com/aws/amazon-ecs-cli(fix-logger)$ ecs-cli compose --debug up 
DEBU[0000] Parsing the compose yaml...                  
ERRO[0000] Unable to open ECS Compose Project            error="open docker-compose.yml: no such file or directory"
FATA[0000] Unable to create and read ECS Compose Project  error="open docker-compose.yml: no such file or directory"

```

Also tried it for `ecs-cli compose --debug up` and it works, but output was very...verbose :)

**Pull request merged**